### PR TITLE
v1

### PR DESCRIPTION
**Pull request by MicheleBertoli**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -40,13 +40,29 @@ $ npm install react-only-if --save
...
 condition   | func    | The condition function. It receives props, context and state.
 Placeholder | element | (optional) The component to render when the condition is false.
 
+### Note for version 0.x users
+
+Following a discussion in [#2](https://github.com/MicheleBertoli/react-only-if/pull/2#issuecomment-241388231),
+the library has been recently rewritten (thanks [Frederik](https://github.com/m90)).
+
+The version 1.x introduces some breaking changes in order to enforce consistency for stateless functional components
+and to make the library compatible with [recompose](https://github.com/acdlite/recompose).
```
Comment (m90) [inline README.md]: While this is true, `recompose` is not the only library that is using this approach. Maybe something a little more generic would be better: 

```
and to make the library play nicely when using functional composition on multiple higher order components.
```

**Comment by m90** [comment]
Nice, other than my nitpick +1, let's release this!

**Comment by m90** [comment]
Amazing, will ship this to production this afternoon! Thanks for all the feedback.

**Comment by MicheleBertoli** [comment]
thank you very much, @m90

**Pull request merged**